dev-php/awl/awl-0.59.ebuild

```diff
@@ -1,18 +1,18 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Sorry to say, but EAPI bump requires a revbump. I suggest you to just leave this file alone.

Reply:
Or just update LICENSES, or remove this after bump to latest?

Reply:
I will just leave it at EAPI 6 and remove it soon, if no complaints about the new version are comming in.